trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = @nameable.correct_name
+    name.length > 10 ? name[0, 10] : name
+  end
+end
```

Comment:
- [ ]  @BenMKT Great work adding the class to trim the name, however, you forgot to import the decorator class before the trimmer decorator class inherits from it. 
- [ ] Also your correct_name method won't trim the name to a `maximum of 10 characters`, this is because of using a range of 0, 10 which is 11 characters(the zero character gets counted).

```suggestion
class TrimmerDecorator < Decorator
  def correct_name
    name = @nameable.correct_name
    name.length > 10 ? name[0, 9] : name
  end
end
```
-

Reply:
- Import Decorator to resolve bug